pytestqt/plugin.py

```diff
@@ -414,6 +417,11 @@ def pytest_addoption(parser):
     parser.addini('qt_no_exception_capture',
                   'disable automatic exception capture')
 
+    parser.addoption('--no-qt-log', dest='qt_log', action='store_false',
+                     default=True)
+    parser.addoption('--qt-log-format', dest='qt_log_format',
+                     default='{rec.type_name}: {rec.message}')
```

Comment:
Maybe those should also be .ini options? i.e. why is `qt_no_exception_capture` a config option, but `--no-qt-log` a commandline flag?